src/app/app.component.ts

```diff
@@ -21,5 +21,5 @@ export class AppComponent {
   Both Mozilla and Internet Explorer have included ways to make sections of a page editable. IE 5.5 had the MSHTML Editing Platform (archive.org), and Mozilla has its Rich Text Editing API. Both technologies allow Web developers to make parts of a page editable -- in slightly different ways, of course.
   Most current in-browser WYSIWYG editors use the third option.`;
   // tslint:enable:max-line-length
-  public name = new FormControl('some text');
+  public name = new UntypedFormControl('some text');
```

Comment:
We can use `FormControl` directly here because it will infer the derived type (string) automatically.

Reply:
thx for checking! it came from an automatic migration